2021-04-08 22:57:17,995 INFO    MainThread:4604 [wandb_setup.py:_flush():69] setting env: {}
2021-04-08 22:57:17,996 INFO    MainThread:4604 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-08 22:57:17,996 INFO    MainThread:4604 [wandb_init.py:_log_setup():336] Logging user logs to C:\Users\aleja\Desktop\Tareas\Reconocimiento Visual con Deep Learning\Tarea1\Git\cc7221\tarea1\src\wandb\run-20210408_225717-2bgyaavz\logs\debug.log
2021-04-08 22:57:17,997 INFO    MainThread:4604 [wandb_init.py:_log_setup():337] Logging internal logs to C:\Users\aleja\Desktop\Tareas\Reconocimiento Visual con Deep Learning\Tarea1\Git\cc7221\tarea1\src\wandb\run-20210408_225717-2bgyaavz\logs\debug-internal.log
2021-04-08 22:57:17,998 INFO    MainThread:4604 [wandb_init.py:init():369] calling init triggers
2021-04-08 22:57:17,998 INFO    MainThread:4604 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-08 22:57:17,999 INFO    MainThread:4604 [wandb_init.py:init():418] starting backend
2021-04-08 22:57:18,000 INFO    MainThread:4604 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn, using: spawn
2021-04-08 22:57:18,007 INFO    MainThread:4604 [backend.py:ensure_launched():123] starting backend process...
2021-04-08 22:57:18,086 INFO    MainThread:4604 [backend.py:ensure_launched():127] started backend process with pid: 7088
2021-04-08 22:57:18,087 INFO    MainThread:4604 [wandb_init.py:init():423] backend started and connected
2021-04-08 22:57:18,088 INFO    MainThread:4604 [wandb_init.py:init():463] updated telemetry
2021-04-08 22:57:18,088 INFO    MainThread:4604 [wandb_init.py:init():482] communicating current version
2021-04-08 22:57:19,072 INFO    MainThread:4604 [wandb_init.py:init():487] got version response 
2021-04-08 22:57:19,072 INFO    MainThread:4604 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2021-04-08 22:57:19,454 INFO    MainThread:4604 [wandb_init.py:init():520] starting run threads in backend
2021-04-08 22:57:19,600 INFO    MainThread:4604 [wandb_run.py:_console_start():1524] atexit reg
2021-04-08 22:57:19,601 INFO    MainThread:4604 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.WRAP
2021-04-08 22:57:19,601 INFO    MainThread:4604 [wandb_run.py:_redirect():1414] Wrapping output streams.
2021-04-08 22:57:19,601 INFO    MainThread:4604 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-08 22:57:19,602 INFO    MainThread:4604 [wandb_init.py:init():544] run started, returning control to user process
2021-04-08 22:57:19,602 INFO    MainThread:4604 [wandb_config.py:__setitem__():141] config set learning_rate = 0.01 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001BFD6790A30>>
2021-04-08 22:57:19,602 INFO    MainThread:4604 [wandb_run.py:_config_callback():749] config_cb learning_rate 0.01 None
2021-04-08 22:57:24,258 INFO    MainThread:4604 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-08 22:57:24,258 INFO    MainThread:4604 [wandb_run.py:_restore():1466] restore
2021-04-08 22:57:24,530 INFO    MainThread:4604 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 938
  total_bytes: 938
}

2021-04-08 22:57:25,516 INFO    MainThread:4604 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 938
  total_bytes: 1965
}

2021-04-08 22:57:25,948 INFO    MainThread:4604 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1967
  total_bytes: 1967
}

2021-04-08 22:57:26,074 INFO    MainThread:4604 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1967
  total_bytes: 1967
}

2021-04-08 22:57:26,204 INFO    MainThread:4604 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1967
  total_bytes: 1967
}

2021-04-08 22:57:26,321 INFO    MainThread:4604 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1967
  total_bytes: 1967
}

2021-04-08 22:57:26,438 INFO    MainThread:4604 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1967
  total_bytes: 1967
}

2021-04-08 22:57:27,587 INFO    MainThread:4604 [wandb_run.py:_show_files():1838] logging synced files
